diff -Naur htop-0.8.3/configure htop-0.8.3wybox/configure
--- htop-0.8.3/configure	2009-06-23 15:53:37.000000000 +0200
+++ htop-0.8.3wybox/configure	2010-06-24 01:20:47.173697784 +0200
@@ -21818,7 +21818,7 @@
   test "$cross_compiling" = yes &&
   { { echo "$as_me:$LINENO: error: cannot check for file existence when cross compiling" >&5
 echo "$as_me: error: cannot check for file existence when cross compiling" >&2;}
-   { (exit 1); exit 1; }; }
+   { (true); true; }; }
 if test -r "$PROCDIR/stat"; then
   eval "$as_ac_File=yes"
 else
@@ -21845,7 +21845,7 @@
   test "$cross_compiling" = yes &&
   { { echo "$as_me:$LINENO: error: cannot check for file existence when cross compiling" >&5
 echo "$as_me: error: cannot check for file existence when cross compiling" >&2;}
-   { (exit 1); exit 1; }; }
+   { (true); true; }; }
 if test -r "$PROCDIR/meminfo"; then
   eval "$as_ac_File=yes"
 else
