Allow adding virtual users with uid=0 and gid=0
diff -Naur pure-ftpd-1.0.35/src/pure-pw.c pure-ftpd-1.0.35wybox/src/pure-pw.c
--- pure-ftpd-1.0.35/src/pure-pw.c	2011-10-31 16:33:57.000000000 +0100
+++ pure-ftpd-1.0.35wybox/src/pure-pw.c	2012-03-08 20:08:25.693229903 +0100
@@ -759,10 +759,10 @@
         return PW_ERROR_MISSING_PASSWD_FILE;
     }
 #ifndef ACCEPT_ROOT_VIRTUAL_USERS
-    if (pwinfo.uid <= (uid_t) 0 || pwinfo.gid <= (gid_t) 0) {
+    /*if (pwinfo.uid <= (uid_t) 0 || pwinfo.gid <= (gid_t) 0) {
         fprintf(stderr, "You must give (non-root) uid and gid\n");
         return PW_ERROR_USERADD_NOT_ROOT;
-    }
+    }*/
 #endif
     if (pwinfo.home == NULL) {
         fprintf(stderr, "Missing home directory\n");        
